src/components/dialogmote/referat/Referat.tsx

```diff
@@ -62,15 +63,20 @@ export const valideringsTexts = {
   konklusjonMissing: "Vennligst angi konklusjon",
   arbeidstakersOppgaveMissing: "Vennligst angi arbeidstakerens oppgave",
   arbeidsgiversOppgaveMissing: "Vennligst angi arbeidsgiverens oppgave",
+  begrunnelseEndringMissing:
+    "Vennligst angi årsaken til at referatet må endres",
 };
 
+export type ReferatMode = "NYTT" | "ENDRET";
```

Comment:
Bytte til enum om den skal brukes til å sendes inn paramteter?

Reply:
Enum funker også, men tenkte det var like greit med string literal her. Får uansett autocomplete osv der den brukes så usikker på om enum gir noe ekstra i dette tilfellet. Se https://szaranger.medium.com/whats-an-enum-65db967d8bb6 f.eks.

Reply:
Tenker du da at det ikke er en fordel med gjenbruk på strings her?
Her vil man jo gjennta den samme stringen ettersom den ikke er lagret i en variabel.

Reply:
Ser forøvrig at i artikkelen står det:
`For scenarios where a reference makes sense rather than a value, and it’s used for conditional logic, an enum can be used.`
Det er kanskje tilfelle her?

Reply:
Mener det er verdien som er nyttig her. I eksempelet med conditional logic i artikkelen (LogLevel) bruker man vel den numeriske enum-verdien (`const num = LogLevel[key]`) og ikke bare strengen, så sånn sett gir det mening med enum der.
Tenker fortsatt det er tilstrekkelig med string literal i tilfellet her, men fordel med enum mtp gjenbruk og slippe å endre flere steder, så kan godt bytte til det.

Reply:
Har fikset det nå!